packages/stream_chat_flutter/CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 🐞 Fixed
 
+- Fix render overflow issue with `StreamTypingIndicator`. It now uses `Flexible` 
+  inside `Row`.
+
+🐞 Fixed
```

Comment:
Hey @juanleondev 👋 

Thank you for contributing to this project 🙂 
Could you please remove this duplicate `Fixed` section?
```suggestion
```

Reply:
Hey @kanat ! Done 😄